build.sbt

```diff
@@ -1231,7 +1221,7 @@ def archetypeProject(archetypeName: String) =
         (unmanagedResources in Compile).value ++ gitIgnoreFiles
       },
       // Don't force copyright headers in Maven archetypes
-      HeaderKey.excludes := Seq("*")
+      excludeFilter in Compile := "*"
```

Comment:
This is too broad, and it has the affect of breaking the archetype generation.

It looks like we don't want to run this plugin at all for the archetype project, so I think you can delete this setting and change `disablePlugins(EclipsePlugin)` on the next line to `disablePlugins(EclipsePlugin, HeaderPlugin)`.

Reply:
Nope, that doesn't work, because this includes `common` and that breaks when the plugin is disabled 😦

Reply:
OK, I think I see the problem... these are _re_-sources, not _sources_, and sbt-header has a separate setting `headerResources`: https://github.com/sbt/sbt-header/blob/master/src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala#L75-L76

I raised a pull request against the sbt-header documentation to make this more obvious: https://github.com/sbt/sbt-header/pull/158

Since I already tested the change locally, I'll go ahead and push to your branch @erip.